d2l-button-icon.html

```diff
@@ -31,6 +31,15 @@
 				border-radius: var(--d2l-button-icon-border-radius);
 				min-height: var(--d2l-button-icon-min-height);
 				min-width: var(--d2l-button-icon-min-width);
+				position: relative;
+			}
+
+			:host([h-align="text"]) button {
+				left: calc(((var(--d2l-button-icon-min-width) - 0.9rem) / 2) * -1);
+			}
+			:host([is-rtl][h-align="text"]) button {
```

Comment:
Did none of the other RTL-detection techniques work? Here's an example that @dbatiste used in search:
https://github.com/BrightspaceUI/search/blob/master/d2l-search-input.html#L38

Reply:
To use, you might need something like:
```
:host-context([dir="rtl"]):host([h-align="text"]) button {
	margin-left: 0.6rem;
	margin-right: 0;
}

:host(:dir(rtl)[h-align="text"]) button {
	margin-left: 0.6rem;
	margin-right: 0;
}
```

I am needing something like this for tabs, and it's working for me so far, but I have not tested across all browsers and permutations for Polymer and shadow/shady DOM.

Reply:
So that didn't work for me but what I did end up getting to work was:
```
:host-context([dir="rtl"])[h-align="text"] button {
    ...
}
```